### PR TITLE
Clean up microdata

### PR DESCRIPTION
**Pull request by yipingliao**
@2is10 Per our earlier discussion.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/libraries/libraryCard.tpl.html [hunk 1]
```diff
@@ -58,7 +60,7 @@
 
           <div ng-switch="clippedDescription" class="kf-keep-lib-description-container">
             <p ng-switch-when="true" class="kf-keep-lib-description">
-              <span ng-bind-html="library.shortDescription"></span>&hellip;
+              <span ng-bind-html="library.shortDescription" itemtype="headline"></span>&hellip;
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryCard.tpl.html]: I’m not sure “headline of the article” captures what this really is. How about `description`?
Danny’s latest designs show the full description, btw.

Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryCard.tpl.html]: Will do. Thanks!

File: kifi-backend/modules/shoebox/angular/src/libraries/libraryCard.tpl.html [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <div class="kf-library-card-header-wrapper" ng-class="{ 'loggedOutUser': isUserLoggedOut, 'library-search-in-progress': librarySearchInProgress, 'library-search': librarySearch }">
-  <div class="kf-library-card-header" ng-class="{ 'loggedOutUser': isUserLoggedOut, 'library-search-in-progress': librarySearchInProgress, 'library-search': librarySearch, 'pageScrolled': pageScrolled }">
+  <div class="kf-library-card-header" ng-class="{ 'loggedOutUser': isUserLoggedOut, 'library-search-in-progress': librarySearchInProgress, 'library-search': librarySearch, 'pageScrolled': pageScrolled }" itemscope itemtype="http://schema.org/CollectionPage">
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryCard.tpl.html:2]: Nice. I think this `itemscope` and `itemtype` should eventually be on a container element that contains the creative work (all of the keep cards), but this is fine for now.

**Comment by 2is10** [comment]
:+1: Huge improvement. Thanks!

**Comment by yipingliao** [comment]
Added a commit for the modal max-height work we just did to this pull request (it was accidentally done on this branch, but the commit message is separate and describes that work).

**Pull request merged**